app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/CoinItem.kt

```diff
@@ -131,6 +133,15 @@ fun CoinItemPreview() {
 @Preview
 fun CoinItemPreviewDark() {
     ComposeTheme(darkTheme = true) {
-        CoinItem()
+        CoinItem(coinItemPreview)
     }
 }
+
+internal val coinItemPreview = CoinItemUiModel(
```

Comment:
Shall we move to `CoinItemUiModel` class. 🤔  As It was called in many places.

Reply:
@Wadeewee it's not production code, I'm still finding a way to manage it better. One of the ways is https://developer.android.com/jetpack/compose/tooling#preview-multipreview 🤔 

Let me try to refactor it in the next PR.